### DISCUSSION TITLE
decreased number of clones in mixcr-4.4.0+

### DISCUSSION THREAD

**Discussion by github-wow** [milaboratory/mixcr#1246] [Q&A] [answered]
Hi,

When I run [one-line `analyze` solution](https://mixcr.com/mixcr/guides/milaboratories-human-bcr-rna-multi/#one-line-solution) with a pair of my files I've noticed that
recent mixcr versions (mixcr-4.3.2, -4.2.0, -4.4.1, -4.4.0) produce .vdjca files with similar sizes:
```
.vdjca        :   1.5 Gb
.refined.vdjca: 560   Mb
```

but the clones-related files are significantly decreased in size (and # of clones)
in versions with [new preset definitions](https://github.com/milaboratory/mixcr/releases/tag/v4.4.0) introduced in v.4.4.0
like "milab-human-rna-tcr-umi-race" instead of deprecated "milab-human-tcr-rna-race-cdr3":

_mixcr-4.4.1, -4.4.0 (with new preset definitions):_

```
.clns           :  48 **Kb**
.clones_TRAD.tsv:  11 **Kb**
```

_mixcr-4.3.2, -4.2.0  (with previous preset definitions):_

```
.clns           :   6   **Mb**
.clones_TRAD.tsv:  27   **Mb**
```

Any help is appreciated.

**Comment by mizraelson** [comment]
Hi,
Can you please share exact commands you used with both version and the length of you reads (R1 and R2)?

**Comment by github-wow** [comment]
Thank you for your reply. The exact lines when done with slurm:
```
mixcr_version="mixcr-4.4.0"

$SCRATCH/mixcr/mixcr -Xmx250g analyze -t 32 milab-human-rna-tcr-umi-race \
      $SCRATCH/input/1.fq.gz                                      \
      $SCRATCH/input/2.fq.gz                                      \
      $SCRATCH/output/${mixcr_version}_test_run.result
```
or
```
mixcr_version="mixcr-4.3.2"

$SCRATCH/mixcr/mixcr -Xmx250g analyze -t 32 milab-human-tcr-rna-race-cdr3 \
      $SCRATCH/input/1.fq.gz                                      \
      $SCRATCH/input/2.fq.gz                                      \
      $SCRATCH/output/${mixcr_version}_test_run.result
```

**Comment by mizraelson** [comment]
and whats the length of R1 and R2?

**Comment by github-wow** [comment]
150+150

**Comment by mizraelson** [comment, marked as answer]
By default,` milab-human-rna-tcr-umi-race` constructs clones based on the `VDJRegion`. This differs from `milab-human-tcr-rna-race-cdr3`, which builds clones utilizing the `CDR3` region. If you want to replicate the previous behavior entirely, you can either continue using the old preset name—which is deprecated but functional—or use the new preset name and add the `--assemble-clonotypes-by CDR3` parameter. 
In your case the 150+150 sequencing coverage is insufficient for spanning the entire VDJRegion. Therefore, ` milab-human-rna-tcr-umi-race` doesn't yield significant results by default.

**Comment by github-wow** [comment]
thank you for the clarification. but despite short coverage it is still sufficient to understand which VDJC genes are used, correct?

**Reply by PoslavskySV** [reply]
Sure, there is still enough information to identify V/D/J/C gene assignment (and you have corresponding columns in the output)

**Comment by github-wow** [comment]
thank you for the clarification
